src/lib/StringUtilities.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Converts strings of the form 'one-two-three' to 'One-Two-Three', also called 'train-case'.
+ */
+export function trainCase(value: string): string {
+    if (value) {
+        let newValue = '';
+
+        for(let i = 0; i < value.length; i++) {
+            newValue += (i === 0 || value[i - 1] === '-') 
+                ? value[i].toUpperCase()
+                : value[i];
+        }
+
+        return newValue;
+    }
+
+    return value;
+}
```

Comment:
Will this work instead? https://lodash.com/docs#kebabCase I know that we are trying to lesson our dependency on lodash, but we can also setup the build to only bring in what we need from lodash.